.travis.yml

```diff
@@ -28,11 +28,13 @@ jobs:
     - stage: deploy
       if: tag IS present
       install:
+        - apt install python3-pyodbc
```

Comment:
What's this for?

Reply:
it has been a test.
travis deploy is failing because of pyodbc in the requirement.txt.
At my local machine I had to install it like this, to make it run. So I tried it out here as well. 
Maybe you know how to fix it correctly.

Reply:
I think it's for the mssql tests. It makes sense I guess.
But... if it's required for master to pass it should already be in there, that's what's confusing to me.

Reply:
Yes. Thought about that as well. But if there are no changes of travis needed because everything should work fine, why it fails on the requirement script?